frontend/appflowy_flutter/lib/mobile/presentation/editor/mobile_editor_screen.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:appflowy/workspace/application/view/view_ext.dart';
+import 'package:appflowy/workspace/application/view/view_service.dart';
+import 'package:appflowy_backend/protobuf/flowy-error/errors.pb.dart';
+import 'package:appflowy_backend/protobuf/flowy-folder2/view.pb.dart';
+import 'package:dartz/dartz.dart' hide State;
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class MobileEditorScreen extends StatefulWidget {
+  static const routeName = '/docs';
+  static const viewId = 'id';
+
+  const MobileEditorScreen({
+    super.key,
+    required this.id,
+  });
+
+  /// view id
+  final String id;
+
+  @override
+  State<MobileEditorScreen> createState() => _MobileEditorScreenState();
+}
+
+class _MobileEditorScreenState extends State<MobileEditorScreen> {
+  late final Future<Either<ViewPB, FlowyError>> future;
+
+  @override
+  void initState() {
+    super.initState();
+
+    future = ViewBackendService.getView(widget.id);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: FutureBuilder(
+          future: future,
+          builder: (context, state) {
+            if (state.connectionState != ConnectionState.done) {
+              return const Center(
+                child: CircularProgressIndicator(),
+              );
+            }
+            if (!state.hasData) {
+              // FIXME: handle the error
+              return const Center(
+                child: FlowyText('No data'),
+              );
+            }
```

Comment:
Maybe we should have a generic design for a failure screen. I saw we have the "success" screen, and I built this in a minute to illustrate what I mean.

<details>
  <summary>Example of error screen</summary>

![image](https://github.com/AppFlowy-IO/AppFlowy/assets/42929161/ee3981ce-47a6-4ef9-be02-4487ef04c29a)

</details>